float32 percent_used
